sentry/src/main/java/io/sentry/transport/AsyncConnection.java

```diff
@@ -14,8 +13,8 @@
 import io.sentry.util.Objects;
 import java.io.Closeable;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
+import java.time.Duration;
+import java.time.Instant;
```

Comment:
this is not Android compatible and cannot live in this package

Reply:
Yes, it's a leftover. Anyway, don't spend too much time on details till it's a draft. The most important part is using the `Connection` as the construct that will be replacable with help of objects like `RateLimiter` and `HttpTransport` would become an implementation detail.